.travis.yml

```diff
@@ -11,7 +11,6 @@ cache:
     - $HOME/.cache # includes bower's cache
 
 env:
-  - EMBER_TRY_SCENARIO=ember-1.13
```

Comment:
I looked into this, but I believe that 1.4.0 is a typo. I think he meant 0.4.0, which is the latest release